backend/src/models/CronJobData.js

```diff
@@ -0,0 +1,49 @@
+import {PocketTransaction} from "./Transaction";
+
+export const BOND_STATUS = {
+    2: "Staked",
+    1: "Unstaking",
+    0: "Unstaked",
+};
+
+export const STAKE_STATUS = {
+    Staked: "Staked",
+    Unstaking: "Unstaking",
+    Unstaked: "Unstaked",
+    "2": "Staked",
+    "1": "Unstaking",
+    "0": "Unstaked",
+};
+
+export class CronJobData {
+
+    /**
+     * @param {string} id The ID of the entity.
```

Comment:
For the params description, use the following format

@param {string} id - The ID of the entity.

Adding the `-` helps separate the property name from the description